pallets/liquidity-mining/src/lib.rs

```diff
@@ -339,7 +347,39 @@ pub mod pallet {
 			);
 			// Create the a pool address with origin and market combo if it doesn't exist
 			let (pool, share_id) = Self::create_pool_account(&market_maker, market);
-			T::OtherAssets::create(AssetId::Asset(share_id), pool.clone(), false, Zero::zero())?;
+			T::OtherAssets::create(share_id, pool.clone(), true, One::one())?;
+			// Transfer existential balance to pool id as fee, so that it never dies
+			T::NativeCurrency::transfer(
+				&market_maker,
+				&pool,
+				T::NativeCurrency::minimum_balance(),
+				ExistenceRequirement::KeepAlive,
+			)?;
+			if let Some(base_asset) = market.base.asset_id() {
+				T::OtherAssets::transfer(
+					base_asset,
```

Comment:
We can't do this. Min balance of BTC might be 1 BTC. That's a weird situation. 

Only PDEX should be used to maintain ED in accounts.

Reply:
One::one() means 1u128 in this context. And later when we will introduce BTC then it will have 18 decimals. So 1 BTC = 10 * 18 u128.